.github/workflows/publish_nightly.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Nightly
```

Comment:
A general query on this file. Does it create a `Nightly` release artifacts from top of `main` branch and replace the old Nightly release artifacts with the newly created ones?

Reply:
Yes, this action will always delete the previous Nightly release and create and publish a new one, so there will always be the latest available.